auth/jwt_controller.go

```diff
@@ -54,6 +77,9 @@ func NewJWTController(model interface{}) *JWTController {
 		model:         model,
 		UsernameField: "username",
 		PasswordField: "password",
+		TokenFunc: func(username interface{}, user interface{}) (string, error) {
```

Comment:
Create controller before returning, then set the `TokenFunc`. That lets you use the controller's struct fields inside the function's scope.

```go
func NewJWTController(model interface{}) *JWTController {
	controller := &JWTController{
		model:         model,
		UsernameField: "username",
		PasswordField: "password"
	}
	controller.TokenFunc = func(r *goyave.Request, user interface{}) (string, error) {
		return GenerateToken(r.String(controller.UsernameField)
	}
	return controller
}